library/validation/LBF_Validation.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+class LBF_Validation{
+
+    /*If another library is used the key names can be modified here*/
+    const VJS_KEY_REQUIRED = 'presence';
+    /*
+ * Function to generate the constraints used in validation.js library
+ * Using the data save in layout options validation
+ */
+    public static function generate_validate_constraints($form_id){
+
+
+        $fres = sqlStatement(
+             "SELECT layout_options.*,list_options.notes as validation_json 
+              FROM layout_options  
+              LEFT JOIN list_options ON layout_options.validation=list_options.option_id
```

Comment:
What I mean is in the ON clause you have to specify the list_id of the list that you are matching on.  Otherwise a different list might have an option_id that matches.